presto-cli/pom.xml

```diff
@@ -170,6 +170,41 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>rpm-maven-plugin</artifactId>
+                <version>2.2.0</version>
+                <executions>
+                    <execution>
+                        <id>generate-rpm</id>
+                        <goals>
+                            <goal>rpm</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <name>presto-cli-rpm</name>
+                    <version>${project.version}</version>
+                    <release>1</release>
```

Comment:
Shouldn't that be Presto version?

Reply:
IIRC it is the version of RPM, for example if you had a bug in RPM scripts you might want to update just new RMP without changing the software version. For example `335-2`.